tests/test_clients/test_clients/test_async_client/test_execute.py

```diff
@@ -0,0 +1,50 @@
+import uuid
+
+import pytest
+from httpx import ConnectError, Request, Response
+
+from botx.clients.methods.v2.bots.token import Token
+from botx.exceptions import BotXConnectError, BotXJSONDecodeError
+
+try:
+    from unittest.mock import AsyncMock
+except ImportError:
+    from unittest.mock import MagicMock
```

Comment:
Mb add some comment that this code here is for 3.7 compatibility

Reply:
Added